.github/workflows/kics.yml

```diff
@@ -0,0 +1,41 @@
+name: kics
+
+on:
+  pull_request:
+    branches: [master]
+    paths:
+      - 'playbooks/**'
+      - 'roles/**'
+      - '.github/workflows/kics.yml'
+  push:
+    branches: [master]
+    paths-ignore:
+      - "docs/**"
+      - "*.md"
+      - .github/workflows/docs.yml
+      - .pre-commit-config.yaml
+
+permissions:
+  pull-requests: write
+  contents: write
```

Comment:
are those still required with comments disabled?

Reply:
Ahh right let me test not sure - probably contents still needs to be there to upload sarif.

Reply:
not permission required